scripts/build.py

```diff
@@ -200,35 +201,10 @@ def process_html(source_path, target_path, file_hashes):
         filepath_with_hash = _insert_hash(filepath, file_hash)
         content = content.replace(filepath, filepath_with_hash)
     content = REMOVE_WS(' ', content)
-    ensure_directory_exists(target_path)
```

Comment:
Why delete?

Reply:
This call was made in each branch of the if ladder that the thread performs now. If it stays here two threads might both find there is no directory, then both will try to create a directory, causing an exception. I moved this call to line 588, which ensures the threads will both have the directory and will not throw exceptions.

Reply:
Hmm, what about the call in line 204?

Reply:
I didn't think about that one before. I had thought that w+ would just truncate anyways so there's not a worry there, but I went back and logged target_path and it's always unique. This actually ends up making sense as the previous issue was on a directory level, not for each file.